test/powershell/enginecore/CommandDiscovery.Tests.ps1

```diff
@@ -0,0 +1,105 @@
+﻿Describe "Command Discovery tests" -Tags "CI" {
+
+    BeforeAll {
+        setup -f testscript.ps1 -content "TestScript"
```

Comment:
I don't think this file is meant to be run, but if it did, it would cause a stack overflow because it's recursive.